6.py

```diff
@@ -0,0 +1,35 @@
+# 15596
+def solve(a):
+    return sum(a)
+
+# 4673
+# set의 특성은 '중복 숫자가 없다' -> 중복 제거하기 위한 필터 역할로 종종 사용 가능
+# set 이용하지 않을 때(리스트형으로 받아 주고 마지막 for문을 통해 중복 여부 확인)
+def d(n):
+    a = 0
+    for x in list(str(n)):
+        a = a + int(x)
+    return int(n) + a
+a= []
+for i in range(1,10001):
+    k = d(i)
+    a.append(k)
```

Comment:
리스트 컴프리헨션으로 사용하면 더 깔끔한 파이썬 코드가 된답니다~ 😄 
```suggestion
a = [d(i) for i in range(1,10001)]
```